### ISSUE TITLE
Issue with convergence

### ISSUE THREAD

**Issue by dheerajiiitv** [breadbread1984/IRCGAN-tf2#1]
Excellent work on this project and I was replicating paper results using your code. I was running the code exactly as you have mentioned in the repository, but facing issue with convergence. I was training this model for single MNIST data. Can you please specify I have to change any other hyperparameters. Here are some images for your reference, Let me know if you want more details

Here is snapshot of tensorboard for your reference- 
Discriminator loss
![45aee7e8-f7fc-4dae-911e-025ac87de6bc](https://user-images.githubusercontent.com/24246192/148649749-b324cb65-449c-4d6e-9861-4761f66b8da6.jpg)
Generator loss
![9effa057-bd2a-4893-b3e6-55af203f2446](https://user-images.githubusercontent.com/24246192/148649759-3f216f74-f5bc-4dc9-b0b7-1d10ed5e81e5.jpg)
)

**Comment by Mahmood-Hussain** [comment]
@dheerajiiitv can you please upload your requirements file? like versions of TensorFlow and other libraries?
